ci-operator/step-registry/openstack/conf/generateconfig/openstack-conf-generateconfig-ref.yaml

```diff
@@ -53,6 +53,10 @@ ref:
                    to be deployed.
         * 'proxy' - Configure the install-config to use a pre-created restricted network (BYON) and
                    a Cluster Proxy.
+    - name: SIZE_VARIANT
+      default: ""
+      documentation: |-
+        The size of the cluster in one of our supported t-shirt values that is standard across all CI environments.
```

Comment:
Do you think it’s worth mentioning that the only valid value is ‘compact’?

Reply:
yes I'll add it